<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciamento Concessionária</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>DADOS</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Perpetua Titling MT</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>CARRO</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="entradaCARRO"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>Perpetua Titling MT</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>ANO DE FABRICAÇÃO</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="entradaano"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="COR">
               <property name="font">
                <font>
                 <family>Perpetua Titling MT</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>COR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="entradacor"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="CARACTERISTICA">
               <property name="font">
                <font>
                 <family>Perpetua Titling MT</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>CARACTERÍSTICA</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="entradacaracteristica"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="CHASSI">
               <property name="font">
                <font>
                 <family>Perpetua Titling MT</family>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>CHASSI</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="entradachassi"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="VALORINICIAL">
             <property name="font">
              <font>
               <family>Perpetua Titling MT</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>VALOR INICIAL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="entradavalorinicial"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="VALORFINAL">
             <property name="font">
              <font>
               <family>Perpetua Titling MT</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>VALOR FINAL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="entradavalorfinal"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QComboBox" name="entradastatus">
           <item>
            <property name="text">
             <string>Vendido</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A Venda</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnrns">
           <property name="text">
            <string>RECOMENDAÇÕES</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btninserir">
           <property name="font">
            <font>
             <family>Perpetua Titling MT</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 170, 0)</string>
           </property>
           <property name="text">
            <string>INSERIR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabelaDados">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>CARRO</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>FABRICAÇÃO</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>COR</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CARACTERISTICA</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CHASSI</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>VALOR INICIAL</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>VALOR FINAL</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>LUCRO</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SITUAÇÃO</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>STATUS</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="OrdNome">
           <property name="text">
            <string>ORDENAR POR NOME</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OrdLucro">
           <property name="text">
            <string>ORDENAR POR LUCRO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ordVenda">
           <property name="text">
            <string>ORDENAR POR VENDA</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="btnlimpar">
           <property name="text">
            <string>LIMPAR TABELA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnsair">
           <property name="font">
            <font>
             <family>Perpetua Titling MT</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(170, 0, 0)</string>
           </property>
           <property name="text">
            <string>SAIR</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>ESTATÍSTICAS </string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>VALOR INVESTIDO</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua Titling MT</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(0, 199, 0)</string>
      </property>
      <property name="text">
       <string>LUCRO REAL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua Titling MT</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(218, 218, 0)</string>
      </property>
      <property name="text">
       <string>LUCRO PREVISTO</string>
      </property>
     </widget>
     <widget class="QLabel" name="valorinicial">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>50</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua Titling MT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lucroreal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lucroprevisto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>371</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua Titling MT</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(170, 170, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar Dados</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar Dados</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionLimpar">
   <property name="text">
    <string>Limpar Dados</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
